llvm/include/llvm/BinaryFormat/ELFRelocs/AArch64.def

```diff
@@ -136,8 +141,11 @@ ELF_RELOC(R_AARCH64_TLS_DTPREL64,                    0x405)
 ELF_RELOC(R_AARCH64_TLS_TPREL64,                     0x406)
 ELF_RELOC(R_AARCH64_TLSDESC,                         0x407)
 ELF_RELOC(R_AARCH64_IRELATIVE,                       0x408)
+// PAuthABI static relocations: Defined in pauthabielf64,
```

Comment:
`R_AARCH64_AUTH_RELATIVE` is a dynamic relocation - see it defined in "AUTH variant Dynamic Relocations" section https://github.com/ARM-software/abi-aa/blob/main/pauthabielf64/pauthabielf64.rst#auth-variant-dynamic-relocations

Reply:
Fixed now, thanks.